code/src/components/SingleMovieDetails/SingleMovieDetails.css

```diff
@@ -0,0 +1,88 @@
+.movie-details-box {
+    background-color: rgba(0, 0, 0, 0.6);
```

Comment:
I like the transparent dark background in the text box, it makes it more readable! Really nice idea!